src/java.desktop/share/classes/java/beans/Beans.java

```diff
@@ -64,6 +64,22 @@ public Beans() {}
      * <p>
      * Instantiate a JavaBean.
      * </p>
+     * The bean is created based on a name relative to a class-loader.
+     * This name should be a dot-separated name such as "a.b.c".
+     * <p>
+     * The given name can indicate either a serialized object or a class.
+     * We first try to treat the beanName as a serialized object
+     * name then as a class name.
+     * <p>
+     * When using the beanName as a serialized object name we convert the
+     * given beanName to a resource pathname and add a trailing ".ser" suffix.
+     * We then try to load a serialized object from that resource.
+     * <p>
+     * For example, given a beanName of "x.y", Beans.instantiate would first
```

Comment:
```suggestion
     * For example, given a beanName of "x.y", {@code Beans.instantiate} would first
```
Method names should be marked up.

Reply:
done